src/Calendar.jsx

```diff
@@ -168,7 +172,7 @@ let Calendar = React.createClass({
 
     var View = VIEW[view]
       , unit = VIEW_UNIT[view]
-      , todaysDate = new Date()
+      , todaysDate = this.getDateOrCurrent(null)
```

Comment:
I was thinking that when we set a currentDate through props, we want this to be the default value to pick. Hence we would want to replace today's date as well. 

At least, that's what worked the best from a UX perspective in my use case :-)

But happy replace it back to today's date if you think that's the way to go.